konami.js

```diff
@@ -0,0 +1,41 @@
+  var konamiCodePosition = 0;
+document.addEventListener('keydown', function(e) {
+  var allowedKeys = {
+    37: 'left',
+    38: 'up',
+    39: 'right',
+    40: 'down',
+    65: 'a',
+    66: 'b'
+  };
+  var konamiCode = ['up', 'up', 'down', 'down', 'left', 'right', 'left', 'right', 'b', 'a'];
+  var key = allowedKeys[e.keyCode];
+  var requiredKey = konamiCode[konamiCodePosition];
+
+  if (key == requiredKey) {
+
+    konamiCodePosition++;
+  } else {
+    konamiCodePosition = 0;
```

Comment:
if the wrong button is pushed, it returns to the beginning value of the array [0].